docs/developer/transform.rst

```diff
@@ -47,11 +50,13 @@ This call to ``transform`` will add the two variables (or variable views) ``a``
         std::tuple<std::tuple<double, float>, std::tuple<double, double>>>(
         a, b, [](const auto &a_, const auto &b_) { return a_ + b_; });
 
+
 Example 2 
 ~~~~~~~~~
 
 In-place transform with two variables and special unit handling, using the help ``overloaded``, to define an "overloaded" lambda:
-This call to ``transform_in_place``  accepts two variables (or variable views) ``a`` and ``b``.
+
+We also call ``transform_in_place`` which accepts two variables (or variable views) ``a`` and ``b``.
 ``a`` is modified in-place, no new variable is returned.
```

Comment:
This paragraph (now split in two) does not seem to make sense any more? Why replace `This call to` with `We also call`, which breaks the connection/grammar?

Reply:
I agree, coming back to this the paragraph made no sense at all. 
I think this was me jotting down points in note form and not coming back to flesh out the changes

I've reverted the change to the original sentence and appended my clarification about `overloaded` which was the main point